Source/Public/Invoke-DockerPull.ps1

```diff
@@ -1,16 +1,16 @@
 . "$PSScriptRoot\..\Private\Write-PassThruOutput.ps1"
 
 function Invoke-DockerPull {
-    [CmdletBinding()]
+    [CmdletBinding(PositionalBinding = $false)]
     param (
         [Parameter(ValueFromPipelineByPropertyName = $true)]
         [String]
         $Registry = '',
```

Comment:
`Invoke-DockerPush.ps1` and `Invoke-DockerTag.ps1` have `$Registry` reordered as the 2nd parameter. 

Consider applying the same coherency for this cmdlet too?

Reply:
My idea was to only do it for params where a convention already exists as with Dockerfile and build context.